pkg/helm/list.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+)
+
+//ExistRelease returns if a release exists
+func ExistRelease(c *action.List, name string) (bool, error) {
```

Comment:
```suggestion
func ReleaseExists(c *action.List, name string) (bool, error) {
```

Reply:
It's odd that it returns bool and error no? Maybee just return err, and if it's nil, it means it does exist?